gtk4/src/auto/editable.rs

```diff
@@ -25,13 +25,6 @@ glib::wrapper! {
     }
 }
 
-impl Editable {
-    //#[doc(alias = "gtk_editable_install_properties")]
-    //pub fn install_properties(object_class: /*Ignored*/&mut glib::ObjectClass, first_prop: u32) -> u32 {
```

Comment:
That's an interesting one that should probably get some special bindings together with `Editable` subclassing support